modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableSkinUtils.java

```diff
@@ -208,4 +208,10 @@ public static <T> ObjectProperty<ObservableList<T>> itemsProperty(TableViewSkinB
         }
         return null;
     }
+
+    /** returns true if the column resize policy is constrained */
+    public static boolean isConstrainedResizePolicy(Callback<? extends ResizeFeaturesBase,Boolean> x) {
```

Comment:
Minor : Need a space after `,`

Reply:
will do.

I do not like inconsistency: everywhere else in this file there is no space after comma in type parameters, for example
```TableViewSkinBase<?,?,?,?,?>```
so, if one were to use a formatter, it will format to 
```TableViewSkinBase<?, ?, ?, ?, ?>```

which will look weird.

Reply:
> will do.

just this one, though. Please don't reformat any others that you aren't otherwise touching.

And yes, it is an area where we are inconsistent (it varies from file to file and often within the same file).

Reply:
certainly, I am not going to touch the rest of the code.